commit 8c13041452bf5df76bce332a1788bfbc02e6aa82
Author: popcornmix <popcornmix@gmail.com>
Date:   Fri Mar 17 13:40:41 2017 +0000

    config: disable MMC driver temporarily for now.
    
    Currently causes a breakage to sdhost driver. However when MMC is disabled Pi3 wifi will not work

diff --git a/arch/arm/configs/bcm2709_defconfig b/arch/arm/configs/bcm2709_defconfig
index 667755d0d8bc..b6ef78d6af1b 100644
--- a/arch/arm/configs/bcm2709_defconfig
+++ b/arch/arm/configs/bcm2709_defconfig
@@ -1046,8 +1046,6 @@ CONFIG_USB_XUSBATM=m
 CONFIG_USB_GADGET=m
 CONFIG_MMC=y
 CONFIG_MMC_BLOCK_MINORS=32
-CONFIG_MMC_BCM2835=y
-CONFIG_MMC_BCM2835_DMA=y
 CONFIG_MMC_BCM2835_SDHOST=y
 CONFIG_MMC_SDHCI=y
 CONFIG_MMC_SDHCI_PLTFM=y
diff --git a/arch/arm/configs/bcmrpi_defconfig b/arch/arm/configs/bcmrpi_defconfig
index 046f3e8757ef..f2e0a58a96c8 100644
--- a/arch/arm/configs/bcmrpi_defconfig
+++ b/arch/arm/configs/bcmrpi_defconfig
@@ -1053,8 +1053,6 @@ CONFIG_USB_G_HID=m
 CONFIG_USB_G_WEBCAM=m
 CONFIG_MMC=y
 CONFIG_MMC_BLOCK_MINORS=32
-CONFIG_MMC_BCM2835=y
-CONFIG_MMC_BCM2835_DMA=y
 CONFIG_MMC_BCM2835_SDHOST=y
 CONFIG_MMC_SDHCI=y
 CONFIG_MMC_SDHCI_PLTFM=y
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1118_8c13041452bf5df76bce332a1788bfbc02e6aa82.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1118_8c13041452bf5df76bce332a1788bfbc02e6aa82.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
